apps/test/integration/util/runLevelTest.js

```diff
@@ -60,7 +60,7 @@ module.exports = function(testCollection, testData, dataItem, done) {
   var validationCallCount = 0;
 
   // Validate successful solution.
-  var validateResult = function(report) {
+  var validateResult = async function(report) {
```

Comment:
i did a bunch of testing with this, with failing and passing tests, and changing it to async doesn't seem to cause any problems. we already use a callback pattern here with `done()`, so the return value of `validateResult` isn't checked.